internetarchive/openlibrary
# Basic `i18n` syntax additions

Sub-task of #9486.


Fix. Adds basic `i18n` syntax where needed, i.e. to simple text with no variables or included HTML.

### Technical

Separated PR's by `i18n` syntax type for ease of review, this one should only include additions of basic `$_(" ")` or `$_(' ')` formatting to previously untranslated text.

Also includes removing relevant files from `i18n` exclude list so as to confirm they now pass the `detect-missing-i18n` check.

### Testing

1. Find a newly `i18n`-ed phrase in one of the files, note that basic on-screen text should use `$_(" ")` syntax, while titles/alts/placeholders should use `$_(' ')` syntax
2. Check the `messages.pot` file to confirm that the phrase now appears in the template
3. Run the `i18n` detection script with `pre-commit run detect-missing-i18n  --all-files`
4. No errors or warnings should appear, and the check should pass

### Screenshot


### Stakeholders

@mekarpeles @cdrini @pidgezero-one
